examples/2d/contributors.rs

```diff
@@ -0,0 +1,315 @@
+use bevy::prelude::*;
+use rand::{prelude::SliceRandom, Rng};
+use std::{
+    collections::BTreeSet,
+    io::{BufRead, BufReader},
+    process::Stdio,
+};
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(velocity_system.system())
+        .add_system(move_system.system())
+        .add_system(collision_system.system())
+        .add_system(select_system.system())
+        .run();
+}
+
+type Contributors = BTreeSet<String>;
+
+struct ContributorSelection {
+    order: Vec<(String, Entity)>,
+    idx: usize,
+}
+
+struct SelectTimer;
+
+struct ContributorDisplay;
+
+struct Contributor {
+    color: [f32; 3],
+}
+
+struct Velocity {
+    translation: Vec3,
+    rotation: f32,
+}
+
+const GRAVITY: f32 = -9.821 * 100.0;
+const SPRITE_SIZE: f32 = 75.0;
+
+const COL_DESELECTED: Color = Color::rgb_linear(0.03, 0.03, 0.03);
+const COL_SELECTED: Color = Color::rgb_linear(5.0, 5.0, 5.0);
+
+const SHOWCASE_TIMER_SECS: f32 = 6.0;
```

Comment:
can we make this something like 3 seconds? i dont want people to get bored too quickly

Reply:
Sure thing, that's why I made it a `const`! Will push a quick patch!